Assets/MixedRealityToolkit/Utilities/MathUtilities.cs

```diff
@@ -49,10 +49,10 @@ public static Vector3 TransformVectorFromTo(Transform from, Transform to, Vector
         /// Retrieve angular measurement describing how large a sphere or circle appears from a given point of view.
         /// Takes an angle (at given point of view) and a distance and returns the actual diameter of the object.
         /// </summary>
-        public static float AngularScaleFromDistance(float angle, float distance)
+        public static float ScaleFromAngularSizeAndDistance(float angle, float distance)
```

Comment:
If possible, please re-add a `public static float ScaleFromAngularSizeAndDistance` with the `Obsolete` tag (can just call directly into this method), so we don't break people currently calling this public method.